packages/itwinui-react/src/core/Table/Table.test.tsx

```diff
@@ -2679,6 +2679,79 @@ it('should handle resize with touch', () => {
   expect(headerCells[2].style.width).toBe('100px');
 });
 
+it.each(['px', '%', 'em', 'rem', 'vw', 'vh', 'ch', 'ex'])(
```

Comment:
no need for all of these imo. Just two or three should be enough: `px`, `%`, `rem`

Reply:
Done (https://github.com/iTwin/iTwinUI/pull/1799/commits/2f9f9613f8a6b39b4bfe76bb2c683c0579941a6e)